### DISCUSSION TITLE
[Pass Through] How correctly pass event handlers?

### DISCUSSION THREAD

**Discussion by TeoDar** [primefaces/community#371] [PrimeVue] [answered]
Good day, PrimeVue team.
I would like to set a custom right-click context menu on column headers. 
But I don’t know how to correctly forward the event in pt to the column element (and whether I really need column). Here's my try: (https://stackblitz.com/edit/flbaqa-89zqsk?file=src%2FApp.vue).
I tested it on the div element above and it works. But how can I implement similar behavior for column headers????

![image](https://github.com/primefaces/community/assets/40923004/b27038a9-323e-48e1-b241-1af1ab9d2487)

In the future, I planned by clicking on RMB, to open the [Overlay Panel](https://primevue.org/overlaypanel/) with a set of buttons...

**Comment by mertsincan** [comment, marked as answer]
Hello @TeoDar,

Please try `onContextmenu`; https://stackblitz.com/edit/flbaqa-nuz1j7?file=src%2FApp.vue

Vue converts `@x` to `onX`. Details; https://vuejs.org/api/render-function.html#mergeprops

**Reply by TeoDar** [reply]
Thanks a lot for solution)
